surf/settings/base.py

```diff
@@ -98,7 +99,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [os.path.join(BASE_DIR, 'templates')],
+        'DIRS': [os.path.join(BASE_DIR, 'apps/filters/templates')],
```

Comment:
Remove to fix templates probably? (but it would break the MPTT templating you made)